packages/layout/src/components/ResizableColumns/types.ts

```diff
@@ -0,0 +1,62 @@
+import { ReactNode } from 'react';
+
+export interface ContainerProps {
+    isResizing?: boolean;
+}
+
+export interface DividerProps {
+    isResizing?: boolean;
+    offset?: number;
+    hideDivider?: boolean;
+}
+
+export interface ColumnProps {
+    columnWidth?: number;
+    minWidth?: string;
+    initialDividerWidth?: number;
+}
+
+export type Width = {
+    px?: number;
+    percent?: number;
+};
+
+type OnResizeParams = {
+    dividerWidth?: number;
+};
+
+export interface ResizableColumnsProps {
+    /**
+     * Sets the initial width for the main column.
+     *
+     * By default the main column is the left one. Pass a value less than zero to set the right column
+     * as the main one.
+     */
+    initialDividerWidth?: number;
```

Comment:
we talk about this and we will use `initialDividerPosition`, then in the description explain that when passing a positive number then it is the divider position respect to the left, and when the number is less than zero then it is the divider position respect to the right